raft-tech/TANF-app
# 3243 promtail pipeline

## Summary of Changes
Pull request closes #3243 

## changes
Added new tags to the dashboard. See below:

![Screenshot 2024-11-20 at 3 45 15 PM](https://github.com/user-attachments/assets/e3386716-fd3b-4caa-897b-c79409b422f7)

![Screenshot 2024-11-20 at 3 45 37 PM](https://github.com/user-attachments/assets/ceefee84-ad97-4165-9a74-0f99ed73b94c)

![Screenshot 2024-11-20 at 3 45 49 PM](https://github.com/user-attachments/assets/33c0961f-39d6-4a5b-aff8-2794bab4970f)


## How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
task up
```

1. Open http://localhost:3000/ and sign in.
1. Login to grafana
2. go to dashboard and explore to see the changes at shown in the screenshots above

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
+ [x] able to filter backend logs by logging level
+ [x] backend logs and services have tags configured

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
